src/components/Testimonials/Testimonials.css

```diff
@@ -1,69 +1,108 @@
 @import url('https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css');
 @import url('https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css');
-.slick-next:before {
-  color:#000 !important;
-  opacity: 1 !important;
+
+:global {
+  .slick-prev:before, .slick-next:before {
+    opacity: 1;
+    color: #007bff;
+    font-size: 25px;
+  }
+  .slick-next {
+    right: -36px;
+  }
+  .slick-prev {
+    left: -41px;
+  }
+  .slick-slide img {
+    display: inline-block;
+  }
+  .slick-slide {
+    position: relative;
+  }
+  .slick-track{
+    display: flex;
+    .slick-slide{
+      display: flex;
+      min-height: 460px;
+      height: 100%;
+      flex: 0 1 328px;
+    }
+  }
+}
+
+* {
+  min-height: 0;
+  min-width: 0;
 }
-.slick-prev:before {
-  color:#000 !important;
-  opacity: 1 !important;
+.alignBottom {
+  position:absolute;
+  bottom:10px;
+  margin: auto;
+  left: 0;
+  right: 0;
 }
 .testimonial{
+  display: -ms-flex;
+  display: -webkit-flex;
   display: flex;
-  flex-wrap: wrap;
-  flex-direction: row;
-  justify-content: center;
   width: 100%;
   margin-top: 20px;
+  position: relative;
 }
-
 .item {
-  position: relative;
   display: flex;
   flex-direction: column;
   align-items: stretch;
   text-align: center;
   flex: 0 1 240px;
   margin-left: 25px;
-  margin-right: 20px;
   margin-bottom: 20px;
   margin-top:20px;
-  border: 2px solid #00ab6b;
+  border: 2px solid #007bff;
   border-radius: 10px;
-  padding-top:10px;
 }
-
 .item h3 {
   font-size: 20px;
   padding-top:20px;
 }
-
 .testimonialImage {
   align-items: center;
   margin-top:20px;
 }
 .slick-slide img{
   display: inline-block !important;
-}
-.description{
-
 }
 .description small {
   clear:both;
 }
 .quote {
   margin-top:10px;
 }
+.quote p{
+  text-indent: 10px;
+}
 .quote:before{
-  content: "\201C"; /*Unicode for Left Double Quote*/
-
-  /*Font*/
+  content: "\201C";
```

Comment:
you think it's better to use `:before` instead of just adding that character in the html of the component?